sitewit/services.py

```diff
@@ -87,6 +87,8 @@ def create_account(self, url, user_id, user_name, user_email,
                 owned by existing user.
             remote_subpartner_id (str, optional): user's partner_id on
                                                   customer side)
+            partner_package (str, optional): subscription level of your
```

Comment:
`package subscription of account's owner`?  And maybe we can call it user_package?

Reply:
But it will create a discrepancy with the API field 😕

Reply:
We already have it with some of the current params.

Reply:
IMO it's unclear what partner package is.